### DISCUSSION TITLE
Couple ideas about the updated Party plugin

### DISCUSSION THREAD

**Discussion by MikailIn** [runelite/runelite#15092] [Ideas]
I'll cut straight to the chase:

1. At ToB Sotetseg; The spec counter does not reset after each maze, which makes this plugin useless there. So if you could get it to reset at start/end of each maze that'd help a lot. Same thing for Hardmode ToB Bloat (this is not a thing for regular ToB Bloat I believe), his defense resets after each stomp so if the plugin could reset with that it'd be wonderful.
2. Again at ToB Sotetseg; Shift + left click pings between shadow realm and the overworld doesn't exist (during maze). This would be useful for the maze runner to ping the first tile ASAP for during speedruns.
3. Would like to see a veng overhead on teammates if they're venged so we don't waste veng on them (you could also integrate this into the party plugin sidepanel, but overhead is the most important part).

And finally the biggest downside of these 2 plugins: It's a hassle to set up if everyone isn't on the same page. That means everyone is using runelite and everyone is familiar with the Party plugin + turn on the spec counter plugin and/or want to set it up quickly. I would love to use the features of this plugin but unless I'm raiding with friends, I can't see randoms cooperating with me (even friends were like naah let's just go in idc). And besides in Hardmode ToB half of the players are on cheat clients, so they can't join the Party even if they wanted to.

A work around for this problem would be automatically forming a party with everyone around you once you enter certain places. ToB and CoX shouldn't be hard to setup I think, Corp might need some tinkering since you leave the cave to get spec back. In addition to this you'd also want to not require the spec counter plugin for person A just so person B can see if A has hit their spec.

I don't think any of these ideas are too "OP" and/or against the rules and I hope it's not super hard to make these a reality :)

Edit: Now that I think about it, just seeing the overheads for specs for everyone around you regardless of party would be even better. I don't know where Jagex would stand on if this is too overpowered or not tho.

**Comment by hex-agon** [comment]
Just a comment on automaking parties, it is something that'll never happen due to server resources involved in making one.
